samples/demo/src/api/todo.ts

```diff
@@ -70,17 +70,30 @@ export const register = (app: express.Application, factory: RepositoryFactory) =
         }
     });
 
+    // Get the failure rate from the environment variable RADIUS_DEMO_FAILURE_RATE. This is a percentage that defaults to 100
+    const radiusDemoFailureRate = parseFloat(process.env.RADIUS_DEMO_FAILURE_RATE || '100') / 100;
+
     app.post(`/api/todos`, async (req, res) => {
         const respository = await factory.create()
         try {
             const item = req.body as Item;
-            const updated = await respository.create(item);
 
+            // Titles that end with a space are used to simulate an error condition and return a 500.
+            if (item && item.title && item.title.endsWith(' ')) {
```

Comment:
Would it be possible to make this behavior opt-in via an environment variable? I'm hesitant to return a 500 error for spaces at the end of an item unless the end-user opts into this behavior. I can foresee this causing a lot of issues when end-users unintentionally add a space.

Reply:
Added RADIUS_DEMO_FAILURE_ENABLED (0=disabled, >0=enabled)